src/main/ml-config/security/amps/generateLayerDescriptor.json

```diff
@@ -1,7 +1,7 @@
 {
     "namespace": "",
     "local-name": "generateLayerDescriptor",
-    "document-uri": "/ext/serviceModel.sjs",
+    "document-uri": "/marklogic-geo-data-services/serviceLib.sjs",
```

Comment:
I renamed serviceModel.sjs to serviceLib.sjs to avoid the goofiness of modelService.sjs depending on a module named serviceModel.sjs.